src/GrpcProtos/capture.proto

```diff
@@ -107,7 +107,9 @@ message CaptureOptions {
   }
 
   ThreadStateChangeCallStackCollection
-      thread_state_change_call_stack_collection = 21;
```

Comment:
Just making sure: We don't serialize the CaptureOptions anywhere? In presets maybe? Is it save to rename things here?

Reply:
If we only use binary serialization this is fine. However, if we use text format protos (or json) this will break things badly.

Reply:
As discussed offline -presets don't store capture options ....